executor/engine.go

```diff
@@ -36,21 +36,20 @@ type Engine interface {
 	// configures the build for execution.
 	CreateBuild(context.Context) error
 	// PlanBuild defines a function that
-	// prepares the build for execution.
+	// handles the resource initialization process
+	// build for execution.
```

Comment:
I think this comment needs to be updated 👍 

```suggestion
	// for the build.
```